tox.ini

```
@@ -2,8 +2,8 @@
 min_version = 4.0
 
 envlist =
-    py{38,39,310,311}-django{3.2,4.1}-wagtail{4.1,5.0,5.1,main}
-    py{39,310,311}-django{4.2}-wagtail{5.0,5.1,main}
+    py{38,39,310,311}-django{3.2,4.1}-wagtail{4.1,5.0,5.1,5.2,main}
+    py{39,310,311}-django{4.2}-wagtail{5.0,5.1,5.2,main}
```

Not sure why the test isn't failing but Django 3.2 doesn't really include Python 3.11.
Also, it might be good to drop Wagtail 5.0, since this has reached EOL (this is a Torchbox-maintained repo, so it should be OK)

```suggestion
    py{38,39,310}-django{3.2,4.1}-wagtail{4.1,5.1,5.2}
    py311-django4.1-wagtail{4.1,5.1,5.2}
    py311-django4.2-wagtail{5.1,5.2}
```